vcd/datasource_vcd_edgegateway.go

```diff
@@ -51,6 +51,11 @@ func datasourceVcdEdgeGateway() *schema.Resource {
 				Computed:    true,
 				Description: "External network to be used as default gateway. Its name must be included in 'external_networks'. An empty value will skip the default gateway",
 			},
+			"default_network_ip": &schema.Schema{
```

Comment:
Just before merging, let's decide whether it would be helpful to rename `default_network_ip` to `default_external_network_ip`

CC: @dataclouder @vbauzysvmware